stdlib/math/math.go

```diff
@@ -0,0 +1,507 @@
+	flux.RegisterPackageValue("math", "erfc", generateMathFunctionX("erfc", math.Erfc))
+	flux.RegisterPackageValue("math", "erfcinv", generateMathFunctionX("erfcinv", math.Erfcinv))
+	flux.RegisterPackageValue("math", "erfinv", generateMathFunctionX("erfinv", math.Erfinv))
+	flux.RegisterPackageValue("math", "exp", generateMathFunctionX("exp", math.Exp))
+	flux.RegisterPackageValue("math", "exp2", generateMathFunctionX("exp2", math.Exp2))
+	flux.RegisterPackageValue("math", "expm1", generateMathFunctionX("expm1", math.Expm1))
+	flux.RegisterPackageValue("math", "floor", generateMathFunctionX("floor", math.Floor))
+	flux.RegisterPackageValue("math", "gamma", generateMathFunctionX("gamma", math.Gamma))
+	flux.RegisterPackageValue("math", "hypot", generateMathFunctionXY("hypot", math.Hypot, "p", "q"))
+	flux.RegisterPackageValue("math", "j0", generateMathFunctionX("j0", math.J0))
+	flux.RegisterPackageValue("math", "j1", generateMathFunctionX("j1", math.J1))
+	flux.RegisterPackageValue("math", "log", generateMathFunctionX("log", math.Log))
+	flux.RegisterPackageValue("math", "log10", generateMathFunctionX("log10", math.Log10))
+	flux.RegisterPackageValue("math", "log1p", generateMathFunctionX("log1p", math.Log1p))
+	flux.RegisterPackageValue("math", "log2", generateMathFunctionX("log2", math.Log2))
+	flux.RegisterPackageValue("math", "logb", generateMathFunctionX("logb", math.Logb))
+	// TODO: change to max and min when we eliminate namespace collisions
+	flux.RegisterPackageValue("math", "m_max", generateMathFunctionXY("max", math.Max))
+	flux.RegisterPackageValue("math", "m_min", generateMathFunctionXY("min", math.Min))
+	flux.RegisterPackageValue("math", "mod", generateMathFunctionXY("mod", math.Mod))
+	flux.RegisterPackageValue("math", "nextafter", generateMathFunctionXY("nextafter", math.Nextafter))
+	flux.RegisterPackageValue("math", "pow", generateMathFunctionXY("pow", math.Pow))
+	flux.RegisterPackageValue("math", "remainder", generateMathFunctionXY("remainder", math.Remainder))
+	flux.RegisterPackageValue("math", "round", generateMathFunctionX("round", math.Round))
+	flux.RegisterPackageValue("math", "roundtoeven", generateMathFunctionX("roundtoeven", math.RoundToEven))
+	flux.RegisterPackageValue("math", "sin", generateMathFunctionX("sin", math.Sin))
+	flux.RegisterPackageValue("math", "sinh", generateMathFunctionX("sinh", math.Sinh))
+	flux.RegisterPackageValue("math", "sqrt", generateMathFunctionX("sqrt", math.Sqrt))
+	flux.RegisterPackageValue("math", "tan", generateMathFunctionX("tan", math.Tan))
+	flux.RegisterPackageValue("math", "tanh", generateMathFunctionX("tanh", math.Tanh))
+	flux.RegisterPackageValue("math", "trunc", generateMathFunctionX("trunc", math.Trunc))
+	flux.RegisterPackageValue("math", "y0", generateMathFunctionX("y0", math.Y0))
+	flux.RegisterPackageValue("math", "y1", generateMathFunctionX("y1", math.Y1))
+
+	SpecialFns = map[string]values.Function{
+		// float --> uint
+		"float64bits": values.NewFunction(
```

Comment:
Why isn't this directly used as a type conversion? Is it different?

In `typeconv` we do:

```
uint64(v.Float())
```

To convert a float64 to uint64.

Reply:
this returns the underlying binary representation of the float.  float64bits(x) != uint64(x)